service-api/module/Application/config/module.config.php

```diff
@@ -51,6 +51,26 @@
 return [
     'router' => [
         'routes' => [
+            'health_check' => [
+                'type' => Literal::class,
+                'options' => [
+                    'route' => '/health-check',
+                    'defaults' => [
+                        'controller' => Controller\HealthcheckController::class,
+                        'action' => 'healthCheck',
+                    ],
+                ],
+            ],
+            'health_check_service' => [
+                'type' => Literal::class,
+                'options' => [
+                    'route' => '/health-check-service',
```

Comment:
To match [the standards](https://docs.opg.service.justice.gov.uk/documentation/adrs/adr-007.html#context):
```suggestion
                    'route' => '/health-check/service',
```